text/2024-01-27-autohinting.md

```diff
@@ -0,0 +1,266 @@
+# Autohinting
+
+*Or, how to smash a glyph into the pixel grid without any help from the font
```

Comment:
Potentially link glyph to https://rsheeter.github.io/font101/#glyph-ids-and-the-cmap-table to  help readers who are unfamiliar with the distinction between glyphs and things you might naively think are glyphs